src/components/sidebar-collection/sidebar-collection.jsx

```diff
@@ -31,9 +31,11 @@ class SidebarCollection extends PureComponent {
    * Handle drop collection.
    */
   onDrop = () => {
-    const databaseName = this.props.database;
-    const collectionName = this.getCollectionName();
-    this.props.globalAppRegistryEmit('open-drop-collection', databaseName, collectionName);
+    if (!this.isNotWritable()) {
+      const databaseName = this.props.database;
+      const collectionName = this.getCollectionName();
+      this.props.globalAppRegistryEmit('open-drop-collection', databaseName, collectionName);
+    }
   }
```

Comment:
Should we also do this for `onImportData()` directly below?

Reply:
onImportData is emitted and picked up by a separate plugin, which already has the check. It's not ideal but it's not currently broken so I wasn't sure if it made sense to add for symmetry even if it wouldn't matter?

Reply:
No, totally fine, I was just curious :+1: